tests/in/glsl/expressions.frag

```diff
@@ -121,6 +121,13 @@ void testNonImplicitCastVectorCast() {
 float global;
 void privatePointer(inout float a) {}
 
+void ternary(bool a) {
+    uint b = a ? 0 : 1u;
+    uint c = a ? 0u : 1;
+
+    uint nested = a ? (a ? (a ? 0u : 1) : 1u) : 1;
```

Comment:
It'd be a little easier to follow the progress of this expression in the output if you used different values for each constant:

    a ? (a ? (a ? 2u : 3) : 4u) : 5